build-tools-internal/build.gradle

```diff
@@ -293,6 +293,8 @@ dependencies {
   testImplementation buildLibs.spock.junit4
   testImplementation buildLibs.json.assert
   integTestImplementation buildLibs.xmlunit.core
+
+  implementation "org.yaml:snakeyaml:${versions.snakeyaml}"
```

Comment:
I don't think this is needed?

Reply:
I moved it to a dependency constraint instead of an explicit dependency. Jackson declares an elder version as a transitive dependency and I think the constraint is the more appropriate way to declare that.